convert_old_db/new_schema_base.h

```diff
@@ -43,4 +43,9 @@ struct hash<UIDAndCID> {
 };
 }
 
+CURRENT_ENUM(OID, uint64_t){INVALID_COMMENT = 0u};
+CURRENT_ENUM(NID, uint64_t){INVALID_NOTIFICATION = 0u};
+CURRENT_ENUM(ANSWER, int8_t){UNSEEN = 0, CTFO = 1, TFU = 2, SKIP = -1};
```

Comment:
This `enum class` is for Midichlorians only, no need for it in schema and/or convert script.